annotations/views/date_appellation_views.py

```diff
@@ -0,0 +1,74 @@
+from rest_framework import viewsets
+from rest_framework import status
+from rest_framework.response import Response
+
+from annotations.models import DateAppellation, DocumentPosition
+from annotations.serializers import DateAppellationSerializer, DocumentPositionSerializer
+
+class DateAppellationViewSet(viewsets.ModelViewSet):
+    queryset = DateAppellation.objects.all()
+    serializer_class = DateAppellationSerializer
+
+    def create(self, request):
+        data = request.data.copy()
+        position = data.pop('position', None)
+        if 'month' in data and data['month'] is None:
+            data.pop('month')
+        if 'day' in data and data['day'] is None:
+            data.pop('day')
+            
+        data['createdBy'] = request.user.id
+        
+        serializer = DateAppellationSerializer(data=data)
+        serializer.is_valid(raise_exception=True)
+        instance = serializer.save()
+        
+        text_id = serializer.data.get('occursIn')
+        if position:
+            if not isinstance(position, DocumentPosition):
+                position_serializer = DocumentPositionSerializer(data=position)
+                position_serializer.is_valid(raise_exception=True)
+                position = position_serializer.save()
+
+            instance.position = position
```

Comment:
Why not just use `position_serializer.save()` instead of saving it into a variable just to save into another variable.